biz.aQute.bndlib/src/aQute/bnd/osgi/Constants.java

```diff
@@ -486,7 +490,11 @@
 	/*
 	 * Module Section
 	 */
+	String								JPMS_MODULE_INFO							= "-jpms-module-info";
 	String								MODULE_INFO_CLASS							= "module-info.class";
+	String								MODULE_INFO_JAVA							= "module-info.java";
+	String								MODULE_INFO_NAME							= "module-info";
+	String								MODULE_JAVA_BASE							= "java.base";
```

Comment:
I don't see any use of `MODULE_INFO_JAVA`, `MODULE_INFO_NAME`, or `MODULE_JAVA_BASE` in this PR. Perhaps these can be removed?